staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6528,6 +6528,18 @@ type Preconditions struct {
 	UID *types.UID `json:"uid,omitempty" protobuf:"bytes,1,opt,name=uid,casttype=k8s.io/apimachinery/pkg/types.UID"`
 }
 
+// LogStreamType represents the desired log stream type.
+type LogStreamType string
+
+const (
+	// LogStreamTypeStdout is the stream type for stdout.
+	LogStreamTypeStdout LogStreamType = "stdout"
+	// LogStreamTypeStderr is the stream type for stderr.
+	LogStreamTypeStderr LogStreamType = "stderr"
+	// LogStreamTypeAll represents the combined stdout and stderr.
+	LogStreamTypeAll LogStreamType = "All"
```

Comment:
```suggestion
	LogStreamTypeAll LogStreamType = "all"
```

Keep the case consistent.

Reply:
This actually follows @smarterclayton 's advice, please see https://github.com/kubernetes/kubernetes/pull/110794#discussion_r921450973.

BTW I am okay with either of these.

Reply:
Is this case-sensitive or not? I prefer it is case-insensitive in CLI.

Reply:
@SergeyKanzhelev @smarterclayton What are your thoughts on this?

Reply:
Making case-insensitive would be the best option for me. But adding api review:

/cc @kubernetes/api-reviewers

Reply:
I am waiting for advice from reviewers 🧐